x/clp/keeper/msg_server.go

```diff
@@ -349,9 +350,10 @@ func (k msgServer) RemoveLiquidity(goCtx context.Context, msg *types.MsgRemoveLi
 
 func (k msgServer) CreatePool(goCtx context.Context, msg *types.MsgCreatePool) (*types.MsgCreatePoolResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
+
 	// Verify min threshold
-	MinThreshold := sdk.NewUintFromString(types.PoolThrehold)
-	if msg.NativeAssetAmount.LT(MinThreshold) { // Need to verify
```

Comment:
This is where the hardcoded PoolThreshold was being used

Reply:
hi @marsifi  , This number was decided by the product team as the min amount of rowan required to create a pool . At that time we decided not to apply an limits on the external asset